_pages/home.md

```diff
@@ -8,81 +8,63 @@ header:
-</div>
-
-<div class="notice--info" markdown="1">
-  <h2>Conference program</h2>
-
-**April 6, 2021**. In our new [blog post]({{ "/update" | relative_url }}), learn more about the conference program and activities we have planned for you.
-
-</div>
-
-<div class="text-center">
-  <a href="{{ "/news" | relative_url }}">Older News</a>
-</div>
-
 ## Welcome!
 
-Welcome to the 16th conference of the European Chapter of the Association for Computational Linguistics (EACL). EACL 2021 will be held from 19 to 23 of April, 2021. While we were planning to hold the conference in Kyiv, due to the current COVID situation the conference will be held entirely online. At EACL 2021, we have tried to recreate a true conference experience. As conferences go online, we all lack social activities and personal contacts, so important in our professional and scientific lives.
+Welcome to the 17th conference of the European Chapter of the Association for Computational Linguistics (EACL). EACL 2023 will be held from 3 to 5 of May, 2023. As the flagship European conference in the field of computational linguistics, EACL welcomes European and international researchers covering a broad spectrum of research areas that are concerned with computational approaches to natural language.
 
-As in previous years, the program of the conference includes poster sessions, tutorials, workshops, and demonstrations in addition to the main conference. As the flagship European conference in the field of computational linguistics, EACL welcomes European and international researchers covering a broad spectrum of research areas that are concerned with computational approaches to natural language.
+## Important Dates
 
-We will also have many social activities: birds of a feather, affinity groups, mentoring activities, social games, and satellite events, for all five days of the conference.
 
-Read more about the conference program [here]({{ "/update" | relative_url }}).
+Conference: May 3-5, 2023
 
-## Important Dates
 
 <table class="table-dates">
     <tbody>
         <tr>
-            <td>Submission deadline<br/>(<i>long and short papers</i>)</td>
-            <td>Wednesday</td>
-            <td>7&nbsp;October 2020</td>
+            <td>Abstract registration deadline for START direct submissions</td>
+            <td>Thursday</td>
+            <td>13&nbsp;October 2022</td>
+        </tr>
+        <tr>
+            <td>Direct paper submission deadline <br/> (<i>long & short papers</i>)</td>
+            <td>Thursday</td>
+            <td>20&nbsp;October 2022</td>
         </tr>
         <tr>
-            <td>Tutorials and workshops</td>
-            <td>Monday&ndash;Tuesday</td>
-            <td>19&ndash;20&nbsp;April, 2021</td>
-        </tr>  
+            <td>Submission deadline for ARR papers</td>
+            <td>Sunday</td>
+            <td>8&nbsp;January 2023</td>
```

Comment:
May be move this down one row since January 2023 comes after December 2022?

Reply:
This is now moved in both home and call for papers pages.